src/components/Select/index.tsx

```diff
@@ -32,6 +32,9 @@ export interface SelectProps
   label?: string
   value?: SelectOption | null
   style?: React.CSSProperties
+  modifyCustomStyle?: (
```

Comment:
how about `configureStyle` or `overrideStyle`? `CustomStyle` is a new concept here, and maybe we should avoid introducing it?